openshift/jh-odh-apply.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  labels:
+    app: jupyterhub
+  name: jh-odh-apply
```

Comment:
Can you change the name? `jh-odh-apply` does not really make sense as a name for buildconfig